saleor/page/models.py

```diff
@@ -44,12 +44,12 @@ def __repr__(self):
         return "%s(pk=%r, title=%r, page_pk=%r)" % (
             class_.__name__,
             self.pk,
-            self.title,
+            self.page.title,
             self.page_id,
         )
 
     def __str__(self):
-        return self.title
+        return self.page.title
```

Comment:
We shouldn't use relations in functions like `__repr__` or `__str__` because it costs as 1 DB query per call.